screens/LoginSuccessScreen.js

```diff
@@ -54,13 +53,29 @@ class LoginSuccessScreen extends React.Component {
         welcome: 'Welcome,',
         loggedIn: 'Logged in at',
         quickOrder: 'Quick Order',
-        ownerRemark: '\'s Owner'
+        ownerRemark: '\'s Owner',
+        settingAlertText: 'Setting',
```

Comment:
change settingAlertText to clientStatusTitle.
change 'Setting' to 'Please complete the following settings'